examples/server.js

```diff
@@ -89,25 +89,21 @@ var sttAuthService = new AuthorizationV1(
     {
       username: process.env.SPEECH_TO_TEXT_USERNAME, // or hard-code credentials here
       password: process.env.SPEECH_TO_TEXT_PASSWORD,
-      iam_apikey: process.env.SPEECH_TO_TEXT_IAM_APIKEY // if using an RC service
+      iam_apikey: process.env.SPEECH_TO_TEXT_IAM_APIKEY, // if using an RC service
+      url: process.env.SPEECH_TO_TEXT_URL ? process.env.SPEECH_TO_TEXT_URL : SpeechToTextV1.URL
```

Comment:
This `url` field is the Authorization URL not the Speech to Text URL. You should not modify this parameter.

Reply:
This actually works well and modifies the right fields and it generates regional IAM API tokens that work properly. If you look at the instance itself, say sttAuthService, you will see that we effectively modify the target_url and the SDK also updates the authorization url to the correct region (this is a print out of the sttAuthService instance):
```
AuthorizationV1 {
  _options: 
   { qs: {},
     url: 'https://gateway-tok.watsonplatform.net/authorization/api',
     username: '',
     password: '',
     iam_apikey: 'XXXXXXXXX',
     jar: 
      RequestJar {
        _jar: CookieJar { enableLooseMode: true, store: { idx: {} } } },
     rejectUnauthorized: true },
  tokenManager: 
   IamTokenManagerV1 {
     iamUrl: 'https://iam.bluemix.net/identity/token',
     tokenInfo: {},
     iamApikey: 'XXXXXXXXX' },
  target_url: 'https://gateway-tok.watsonplatform.net/text-to-speech/api' }
```

As per the documentation, this is the way they suggest to pass-in the service url parameter (see|[https://github.com/watson-developer-cloud/node-sdk#authorization]).
As mentioned, I tested the tokens generated with the client-side and they fully work, seems to me to be the more clean way to put the parameters. Test for yourself and let me know if you find any issues with this.